nixos/modules/services/web-apps/comfyui.nix

```diff
@@ -0,0 +1,139 @@
+{ config, lib, options, pkgs, ...}:
+
+with lib;
+
+let
+  cfg = config.services.comfyui;
+  defaultUser = "comfyui";
+  defaultGroup = defaultUser;
+  mkComfyUIPackage = cfg: cfg.package.override {
+    modelsPath = "${cfg.dataPath}/models";
+    inputPath = "${cfg.dataPath}/input";
+    outputPath = "${cfg.dataPath}/output";
+    customNodes = cfg.customNodes;
+  };
+in
+{
+  options = {
+    services.comfyui = {
+      enable = mkEnableOption
+        (mdDoc "The most powerful and modular stable diffusion GUI with a graph/nodes interface.");
+
+      dataPath = mkOption {
+        type = types.str;
+        default = "/var/lib/comfyui";
+        description = mdDoc "path to the folders which stores models, custom nodes, input and output files";
+      };
+
+      package = mkOption {
+        type = types.package;
+        default = (
+          if config.cudaSupport
+          then pkgs.comfyui
```

Comment:
```suggestion
          if config.cudaSupport
          then pkgs.comfyui-cuda
```

Reply:
Haven't checked the derivation yet, but even *if* cuda is the default, we should be explicit here.

Reply:
nice catch, I made a typo